usaspending_api/search/delta_models/award_search.py

```diff
@@ -532,6 +534,44 @@
     GROUP BY
         GROUPED_BY_DEFC.award_id
 ) IIJA_DEFC on IIJA_DEFC.award_id = awards.id
+-- Total outlays calculation
+LEFT JOIN (
+    SELECT award_id,
+            COALESCE(total_gross_outlay_amount_by_award_cpe, 0)
+            + COALESCE(total_ussgl487200_down_adj_pri_ppaid_undel_orders_oblig_refund_cpe, 0)
+            + COALESCE(total_ussgl497200_down_adj_pri_paid_deliv_orders_oblig_refund_cpe, 0) AS total_outlays
```

Comment:
I believe using these `COALESCE` statements will conflict with AC #5:
> Total outlays calculation is blank/NULL when there is no outlay data (we do NOT show “0”)

Reply:
At this point, in the outer query that you've commented on, I've already satisficed AC #5. These coalesces are in case any of the columns being added together are null, if even just one column is null the result would be NULL.